### PR TITLE
Gitter to slack URL in issue_template

### PR DESCRIPTION
**Pull request by albertteoh**
Signed-off-by: albertteoh <albert.teoh@logz.io>

## Which problem is this PR solving?
- Relates to https://github.com/jaegertracing/jaeger/issues/2871

## Short description of the changes
- Update chat room link from gitter to slack.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Welcome to the Jaeger project! 👋🎉
 - Please be respectful and considerate of others when commenting on issues.
 - Please provide as much information as possible so we all understand the issue.
 - If you only have a question, you may get a faster response by asking in
-    - our chat room https://gitter.im/jaegertracing/Lobby, or
+    - our chat room https://cloud-native.slack.com/archives/CGG7NFUJ3, or
     - the forum https://groups.google.com/d/forum/jaeger-tracing
```
Comment (yurishkuro) [inline .github/ISSUE_TEMPLATE.md:9]: let's remove the mail list link

**Review by yurishkuro — APPROVED** [review]
I wonder if we could do a single template for the org. And it would be great to switch to Forms, so that this welcome text/comment wouldn't remain in the tickets (people always leave it in).

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (albertteoh) [reply .github/ISSUE_TEMPLATE.md]: done

**Comment by albertteoh** [comment]
> I wonder if we could do a single template for the org. 

Something like [github's community health file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file)?

> And it would be great to switch to Forms

Did you mean [using Google Forms](https://medium.com/@01010111/using-google-forms-to-submit-github-issues-efdb5f876b)?

> so that this welcome text/comment wouldn't remain in the tickets (people always leave it in)

For my understanding, what is the downside with leaving the text/comment in the ticket? Plain-text email clients?

**Comment by yurishkuro** [comment]
1 - yes, having central files / templates in the org's .github, like OTEL does.

2 - no, that would be very odd experience to go to another site to file an issue. I saw a few months back an announcement that GH is going to supports actual forms where the issue template is not just the markdown inserted into description, but a form with separate fields, titles, etc. I don't know if that feature is in private beta though, can't seem to find it in the docs.

**Comment by yurishkuro** [comment]
> For my understanding, what is the downside with leaving the text/comment in the ticket? Plain-text email clients?

that, plus I often find myself editing descriptions (to add formatting, add 'Resolves...'), and the comment text is always there, I don't think anyone ever bothers to remove it.

**Pull request merged**